mlir/lib/Dialect/Rock/Tuning/RockTuningImpl.cpp

```diff
@@ -57,10 +57,10 @@ void createGemmTuningRangeBF(TuningParamSet *newSpace,
   const std::vector<std::vector<uint32_t>> validRangeWmmaGemmParams = {
       {4, 8, 16, 32, 64, 128, 256},
       {16, 32, 64, 128, 256},
-      {16, 32},
+      {1, 2, 4, 8},
       {4, 8, 16, 32, 64, 128},
       {4, 8, 16, 32, 64, 128},
-      {16},
+      {4, 8, 16},
```

Comment:
Will we want kPack < 16?

(I think so but)

Reply:
Yes, for two reasons:
- For fp16 the maximum hardware vector is 8 (128bits)
- In general we saw that there are configurations where a vector smaller than the max allowed  is faster